### PR TITLE
Remove unused OpenALSoundManager::m_can_vorbis and EXT_vorbis check

### PR DESCRIPTION
**Pull request by jayarndt**

### CODE REVIEW

File: src/sound_openal.cpp [hunk 1]
```diff
@@ -295,14 +293,6 @@ class OpenALSoundManager: public ISoundManager
 			return;
 		}
 
-		if(alcIsExtensionPresent(m_device, "EXT_vorbis")){
```
Comment (rubenwardy) [inline src/sound_openal.cpp]: `alcIsExtensionPresent` definitely doesn't have any side effects?

Comment (jayarndt) [inline src/sound_openal.cpp]: OpenAL docs: "This function queries if a specified 
context extension is available"
I'm pretty confident.

**Comment by est31** [comment]
:+1: looks good.

**Comment by nerzhul** [comment]
:+1:

**Comment by paramat** [comment]
:+1: Will merge later.

**Comment by paramat** [comment]
6fead2818acf00a63c89ec613230a59bc0c4cab5

**Pull request closed without merging**